requirements/main.txt

```diff
@@ -9,5 +9,7 @@ fbprophet>=0.5
 scikit-learn>=0.21.2
 pandas>=0.23.1
 tqdm>=4.32.1
-ipywidgets
-testfixtures
\ No newline at end of file
+torch>=1.4.0
+tensorboard>=2.2.1
+ipywidgets>=7.5.1
+testfixtures
```

Comment:
will prevent future issues if we freeze the version... which probably applies to the whole file.